gymnasium/spaces/multi_binary.py

```diff
@@ -90,29 +93,33 @@ def sample(self, mask: Optional[np.ndarray] = None) -> np.ndarray:
 
         return self.np_random.integers(low=0, high=2, size=self.n, dtype=self.dtype)
 
-    def contains(self, x) -> bool:
+    def contains(self, x: Any) -> bool:
         """Return boolean specifying if x is a valid member of this space."""
         if isinstance(x, Sequence):
             x = np.array(x)  # Promote list to array for contains check
 
         return bool(
             isinstance(x, np.ndarray)
             and self.shape == x.shape
-            and np.all((x == 0) | (x == 1))
+            and np.all(np.logical_or(x == 0, x == 1))
```

Comment:
Is this required for typing, and exactly equivalent?

Reply:
I believe this is equivalent and it makes it more clear for users